src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -8,6 +8,8 @@
     <AssemblyVersion Condition="'$(AssemblyVersion)'==''">999.999.999.999</AssemblyVersion>
     <CLSCompliant Condition="'$(CLSCompliant)'=='' and '$(IsTestProject)'=='true'">false</CLSCompliant>
     <CLSCompliant Condition="'$(CLSCompliant)'==''">true</CLSCompliant>
+
+    <ExcludeLocalizationImport Condition="'$(ExcludeLocalizationImport)'=='' And !Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')">true</ExcludeLocalizationImport>
```

Comment:
My thinking is that we can use a convention that when loc is added, it must be done for all languages.  It doesn't have to be translated in all languages initially, of course; the loc files just have to be put in place.